### ISSUE TITLE
Update to use .NET version 5 by default

### ISSUE THREAD

**Issue by cdimitroulas** [SaturnFramework/Saturn.Template#33]
I was going through the Get Started guide for Saturn and noticed that the template uses .NET version 3.1 by default. Would it be better to update this to .NET version 5 at this point?

**Comment by davidtimovski** [comment]
.NET 6 (LTS) at this point. There was kind of an excuse not to upgrade to .NET 5 because it wasn't LTS but now it just seems like lack of interest in maintaining this. Advertises itself as High Performance but no interest in moving to the most performance-focused .NET update. 🤔

**Comment by cdimitroulas** [comment]
@Krzysztof-Cieslak I would be happy to try and open a PR for this as it seems relatively simple (just looking at a [similar PR](https://github.com/SaturnFramework/Saturn.Template/pull/24) for the upgrade to 3.1) if you give me a confirmation that you'd be happy with this change :D

**Comment by cartermp** [comment]
There is .NET 6 work in progress here for Saturn. Unfortunately, it's still going to be a while because there is an F# compiler bug we're waiting on being fixed (should be a month or so). https://github.com/SaturnFramework/Saturn/pull/322

We'll likely just bypass .NET 5 in the template altogether

**Comment by cdimitroulas** [comment]
Thanks @cartermp, closing this in favour of https://github.com/SaturnFramework/Saturn/issues/321 then
